src/Apps/ArtAdvisor/03-Instruction-Experiments/server.ts

```diff
@@ -0,0 +1,215 @@
+import OpenAI from "openai"
+import dotenv from "dotenv"
+import express, { Request, Response } from "express"
+import { instructions } from "./00-prompt-lib/collector-profile-two"
+import chalk from "chalk"
+
+/*
+ * Setup express server, OpenAi client, and config.
+ */
+
+dotenv.config()
```

Comment:
Is this necessary?

Reply:
Nope. Good catch. Left over from the `quantum` code. Have removed in [1ff5da7](https://github.com/artsy/force/pull/13850/commits/1ff5da7b84a00d150884cb5f125e3fe8532844bd).